realm/realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java

```diff
@@ -93,15 +95,13 @@
         this.columnInfo = (AllTypesColumnInfo) columnInfo;
     }
 
-    @Override
```

Comment:
I don't quite understand why the `@Override` annotation is removed. Could you please explain a bit?